configs/quota/config/config.yaml

```diff
@@ -0,0 +1,43 @@
+types:
+- name: ratelimit_by_service_by_consumer
+  dimensions:
+    service: STRING
+    consumer: STRING
+  maxAmount: 500
+  expiration: 1s
+- name: ratelimit_by_service_by_datacenter
+  dimensions:
+    service: STRING
+    datacenter: STRING
+  maxAmount: 10000
+  expiration: 1s
+
+contructors:
+- name: rl_by_service_by_consumer
+  type: ratelimit_by_service_by_consumer
+  dimensions:
```

Comment:
Will we be able to flatten these fields within the constructor ? Or do we need a "struct params"